authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/services/dataoffer/DataSourceAvailabilityMapper.kt

```diff
@@ -0,0 +1,11 @@
+package de.sovity.authorityportal.web.services.dataoffer
+
+import de.sovity.edc.ext.wrapper.api.common.model.DataSourceAvailability
+
+fun DataSourceAvailability.fromString(str: String): DataSourceAvailability {
+    return when (str) {
+        "LIVE" -> DataSourceAvailability.LIVE
+        "ON_REQUEST" -> DataSourceAvailability.ON_REQUEST
+        else -> throw IllegalArgumentException("Unknown availability: $this")
```

Comment:
default to "LIVE", please. remember, it can be null for a Management-API created value.

also fix the nullability, if this `str` was pulled from a uiAsset, we need to be 100% null-friendly. Same with anything else pulled from the UiAssetJson